src/main/frontend/util/marker.cljs

```diff
@@ -30,13 +30,13 @@
 
 (defn header-marker-pattern
   [markdown? marker]
-  (re-pattern (str "^" (when markdown? "#*\\s*") marker)))
+  (re-pattern (str "^" (when markdown? "#+\\s+") marker)))
```

Comment:
@tiensonqin @llcc We don't have tests for this with replace-marker so not as sure about behavior here. Does this seem reasonable?

Reply:
@logseq-cldwalker it makes sense for me. Will write a test about `replace-marker` later.